Makefile

```diff
@@ -202,7 +202,7 @@ bump-copyright:
 $(FMT):
 	wget -qO - 'https://codeload.github.com/fenollp/erlang-formatter/tar.gz/master' | tar xz -C $(ROOT)/make/
 
-fmt: TO_FMT ?= $(shell git --no-pager diff --name-only HEAD origin/master -- "*.erl" "*.hrl" "*.escript")
+fmt: TO_FMT ?= $(shell git --no-pager diff --name-only HEAD origin/master | grep -e  "\.erl\|\.hrl\|\.escript")
```

Comment:
Curious why this change was made?

Reply:
seems related to my git version on my old CentOS 6 dev box.
i'll remove this